test/service/test_azul_webservice.py

```diff
@@ -0,0 +1,180 @@
+import os
+import sys
+import logging
+import requests
+from threading import Thread
+from unittest import TestCase
+from chalice.local import LocalDevServer
+from chalice.config import Config as ChaliceConfig
+from azul import Config as AzulConfig
+
+sys.path.append(os.path.join(AzulConfig().home_directory, 'lambdas', 'service'))
+
+from app import app as service_app
+
+log = logging.getLogger(__name__)
+stream_handler = logging.StreamHandler(sys.stdout)
```

Comment:
Use `logging.basicConfig(level=logging.DEBUG)` instead to configure logging and place it into a `setupModule()` function. WIthout that a mere import of this module could affect the global logging configuration.